src/components/SelectableMainTable.tsx

```diff
@@ -34,20 +35,30 @@ const SelectableMainTable: FC<Props> = ({
   headers,
   ...props
 }: Props) => {
+  const [currentSelectedIndex, setCurrentSelectedIndex] = useState<number>();
   const isAllSelected =
     selectedNames.length === filteredNames.length && selectedNames.length > 0;
   const isSomeSelected = selectedNames.length > 0;
 
+  useEventListener<"mousedown">("mousedown", (event) => {
```

Comment:
this is needed because when holding down shift key and clicking in the browser, the default behaviour is to select all text content between two locations. This prevents the checkbox from being selected while the shift key is held down.

Reply:
I think disabling this so generally is not a good idea, because it prevents text selection. Better pin it down to `event.target` being a relevant checkbox?

Reply:
Right, I see.

Reply:
Maybe add a comment about the text selection is prevented, as by the code alone it is not clear why this special listener is here.